tests/test_func.py

```diff
@@ -691,6 +691,23 @@ def sum_count(result: sum_count_type, val: int) -> sum_count_type:
         assert row["count"] == len(rows) and row["sum"] == 45
 
 
+def test_func_plpy(db: gp.Database):
+    @gp.create_function
+    def test_plpy() -> bool:
+        # Function with name starting with "__func_" will only be created by
+        # exec() when using source serialization, i.e. when dill is not
+        # available on server.
+        #
+        # If dill is used for deserialization, plpy will not be available.
+        return "plpy" in globals().keys() or (
+            len([True for name in globals().keys() if name.startswith("__func_")]) == 0
+        )
```

Comment:
len([True for name in globals().keys() if name.startswith("__func_")]) == 0 -> all([name.startswith("__func_") for name in globals().keys()]) ?